building the documentation fails in vagrant box

Running `make install` for the documentation (as ansible does) fails using the provided Vagrantfile. This is commit 6412a4ab7ad6fac62837d0676b7e7a30e001e5e2, Vagrant 1.9.5 on ArchLinux.

```
(anitya) [vagrant@anitya-dev docs]$ make install
mkdir -p _static
sphinx-build -b html -W -d _build/doctrees   . _build/html
Running Sphinx v1.6.2
/home/vagrant/devel/anitya/lib/versions/rpm.py:48: UserWarning: Failed to import 'rpm', emulating RPM label comparisons
  warnings.warn("Failed to import 'rpm', emulating RPM label comparisons")
[anitya.config INFO] The Anitya configuration file, /etc/anitya/anitya.toml, does not exist.
[anitya.config WARNING] SECRET_KEY is not configured, falling back to the default. This is NOT safe for production deployments!
/home/vagrant/.virtualenvs/anitya/lib/python2.7/site-packages/sphinxcontrib/autohttp/flask.py:23: RemovedInSphinx17Warning: sphinx.util.compat.Directive is deprecated and will be removed in Sphinx 1.7, please use the standard library version instead.
  from sphinx.util.compat import Directive
/home/vagrant/.virtualenvs/anitya/lib/python2.7/site-packages/sphinxcontrib/autohttp/flask_base.py:22: RemovedInSphinx17Warning: sphinx.util.compat.Directive is deprecated and will be removed in Sphinx 1.7, please use the standard library version instead.
  from sphinx.util.compat import Directive

Exception occurred:
  File "/home/vagrant/.virtualenvs/anitya/lib/python2.7/site-packages/sphinxcontrib/autohttp/flask.py", line 46, in setup
    if 'http' not in app.domains:
AttributeError: 'Sphinx' object has no attribute 'domains'
The full traceback has been saved in /tmp/sphinx-err-tHEbwI.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:53: recipe for target 'html' failed
make: *** [html] Error 1
```

--- a/requirements/readthedocs.txt
+++ b/requirements/readthedocs.txt
@@ -14,7 +14,7 @@ flask-wtf
 jinja2 >= 2.4
 python-openid; python_version < '3.0'
 python3-openid; python_version >= '3.0'
-sphinx
+sphinx==1.5.6
 sphinxcontrib-httpdomain
 sqlalchemy >= 0.9
 straight.plugin==1.4.0-post-1

--- a/requirements/test_requirements.txt
+++ b/requirements/test_requirements.txt
@@ -14,5 +14,5 @@ requests-oauthlib
 contextlib2
 
 # Required to test building the docs
-sphinx
+sphinx==1.5.6
 sphinxcontrib-httpdomain

--- a/tox.ini
+++ b/tox.ini
@@ -40,8 +40,7 @@ commands =
 [testenv:docs]
 changedir = docs
 deps =
-    sphinx
-    sphinxcontrib-httpdomain
+    -rrequirements/readthedocs.txt
     -rrequirements/requirements.txt
 whitelist_externals =
     mkdir

